src/js/mictest.js

```diff
@@ -44,16 +44,19 @@ function MicTest(test) {
   for (var i = 0; i < this.inputChannelCount; ++i) {
     this.collectedAudio[i] = [];
   }
+  var AudioContext = window.AudioContext || window.webkitAudioContext;
+  this.audioContext = new AudioContext();
 }
 
 MicTest.prototype = {
   run: function() {
-    if (typeof audioContext === 'undefined') {
-      this.test.reportError('WebAudio is not supported, test cannot run.');
+    this.audioContext.resume().then(function() {
```

Comment:
nit: Why do we need `resume()` here?
Maybe it is worth adding a comment here.

Reply:
https://developers.google.com/web/updates/2017/09/autoplay-policy-changes#webaudio TL;DR After the AudioContext has been created, it has to be resumed after a user has interacted with the page.

Reply:
Added comment.